hack/test-update-storage-objects.sh

```diff
@@ -203,8 +203,8 @@ for test in ${tests[@]}; do
   kube::log::status "Verifying we can retrieve ${resource}/${namespace}${name} via kubectl"
   # We have to remove the cached discovery information about the old version; otherwise,
   # the 'kubectl get' will use that and fail to find the resource.
-  rm -rf ${HOME}/.kube/cache/discovery/localhost_8080/${KUBE_OLD_STORAGE_VERSIONS}
-  ${KUBECTL} get ${namespace_flag} ${resource}/${name}
+  rm -rf "${HOME}"/.kube/cache/discovery/localhost_8080/${KUBE_OLD_STORAGE_VERSIONS}
+  ${KUBECTL} get "${namespace_flag}" "${resource}"/"${name}"
```

Comment:
No need to drop out of quotes for the /
```suggestion
  ${KUBECTL} get "${namespace_flag}" "${resource}/${name}"
```

Reply:
this still needs to be addressed